core/templates/dev/head/components/rule_editor.html

```diff
@@ -17,6 +17,15 @@
           </span>
         </span>
 
+        <span ng-if="item.type == 'checkbox'" style="color: black;">
```

Comment:
This should be encapsulated within an object template (where the purpose of the object is to display a checkbox form).